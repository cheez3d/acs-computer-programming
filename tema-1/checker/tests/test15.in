plamani inima ficat rinichi
77778666#62#222072662227773327777
arterele duc oxigen la plamani
3
arterele venele
oxigen carbon
la dela
ochi nas gene
