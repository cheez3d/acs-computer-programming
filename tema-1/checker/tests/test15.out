755526266444044466444620333444222#2807774446644422244#444
stomac pancreas
8883366335553303882220222#277722666#6603#3355520755526266444
3
66622244444 6627777 4336633 
66622244444 1
0
0
